consts.py

```diff
@@ -0,0 +1,60 @@
+
+# KAFKA TOPICS
+KAFKA_CASE_TOPIC = 'case-sql'
+KAFKA_FORM_TOPIC = 'form-sql'
+KAFKA_LOCATION_TOPIC = 'location'
+
+
+MAX_RECORDS_TO_PROCESS = 50000
+DATA_LAKE_DOMAIN = ['inp-dashboard']
+
+# TODO Split the different forms and cases types to be processed by separate process instances
+ALLOWED_FORMS = [
```

Comment:
I guess this kind of lives in https://github.com/dimagi/dimagi_lake/pull/1/files#diff-3cb9cc54ef7390b6e0cc88ca46bfbba156adb3e08490c21c85763e86600e72e5? any way to reconcile those? (optional)

Reply:
I think this whitelist should contain and use the form xmlns Ids to identify the forms But I think we can convert this to list of tuples `( xmlns_id, form_name)` instead of just the list of form xmlns ids